src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -30,43 +24,31 @@ public LineService(LineRepository lineRepository, StationService stationService)
     }
 
     public LineResponse saveLine(LineRequest request) {
-        Station upStation = stationService.findById(request.getUpStationId());
-        Station downStation = stationService.findById(request.getDownStationId());
-        Line persistLine = lineRepository.save(new Line(request.getName(), request.getColor(), upStation, downStation, request.getDistance()));
-        List<StationResponse> stations = getStations(persistLine).stream()
-                .map(it -> StationResponse.of(it))
-                .collect(Collectors.toList());
-        return LineResponse.of(persistLine, stations);
+        Station upStation = findStationById(request.getUpStationId());
```

Comment:
개인적으로 퍼블릭 메서드 명을 정할때는 협력에 맞게 추상적인 작성하는 것을 선호하는데요.
그 이유는 내부 구현이나 파라미터가 바뀌더라도, 메서드 명을 유지할 수 있기 때문입니다.
메서드 명을 유지한다는 것은 생각보다 단순해보이기도하지만,
처음 정의한 협력을 바꾸지 않는다는 관점에서 바라보면 굉장히 중요하게 느껴지기도 하네요 🤔
개인적인 생각이니 참고만 해주세요 :)

>구현과 관련된 모든 정보를 캡슐화하고 객체의 퍼블릭 인터페이스에는 협력과 관련된 의도만을 표현해야 한다.

```suggestion
        Station upStation = findStation(request.getUpStationId());
```

Reply:
오!! 좋은생각입니다!!!